string screen_line1
string screen_line2
string screen_line3
string screen_line4
